src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.DateTime.netfx.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+
+namespace System.Diagnostics
+{
+    partial class Activity
+    {
+        private DateTime GetUtcNow()
```

Comment:
Place a comment that says  DateTime.UtcNow is only accurate to 16 msec on Desktop, which is why we need to use Stopwatch()